avaframe/com2AB/com2ABCfg.ini

```diff
@@ -0,0 +1,62 @@
+### Config File - This file contains the main settings for the com2AB run
+## Set your parameters
+# This file is part of Avaframe.
+# This file will be overridden by local_com2ABCfg.ini if it exists
+# So copy this file to local_com2ABCfg.ini, adjust your variables there
+
+
+# Required settings------------------------------
+[INPATH]
+ProfileLayer = avaframe/data/avaSlide/LINES/slideProfiles
+DGMSource = avaframe/data/avaSlide/slideTopo.asc
+SplitPointSource = avaframe/data/avaSlide/POINTS/slidePoints
+saveOutPath = avaframe/com2AB/Outputs/
+outputName = SlideProfile3
+
+# ProfileLayer          # YOUR PATH TO AVALANCHE PATH
+# DGMSource             # YOUR PATH TO DEM
+# SplitPointSource      # YOUR PATH TO SPLITPOINT
+# saveOutPath           # YOUR PATH TO OUTPUT FILE
+# outputName            # YOUR OUTPUT FILE NAME
```

Comment:
I'd prefer to have the comments for each variable right above it like:

# YOUR PATH TO AVALANCHE PATH
ProfileLayer = ....

# YOUR PATH TO ....

As far as I'm aware this should be no problem for the the configparser lib. Are there any good reasons to not do it like this?